### PR TITLE
Code cleanup

### PR DESCRIPTION
**Pull request by carlo-dev-git**
Misc coding cleanup

Description
-----------
Misc coding cleanup and typo fixes.

Test Steps
-----------
No steps needed.

Related Issue
-----------
No related issue.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: portable/RVDS/ARM_CM4F/port.c [hunk 1]
```diff
@@ -243,14 +243,14 @@ __asm void vPortSVCHandler( void )
     PRESERVE8
 
     /* Get the location of the current TCB. */
-    ldr r3, = pxCurrentTCB
+    ldr r3, =pxCurrentTCB
     ldr r1, [ r3 ]
     ldr r0, [ r1 ]
     /* Pop the core registers. */
-    ldmia r0 !, {r4-r11,r14}
+    ldmia r0 !, { r4 - r11, r14 }
```
Comment (RichardBarry) [inline portable/RVDS/ARM_CM4F/port.c:250]: "r4-r11" was correct, is specifies registers r4 to r11, not a subtraction.  Likewise for all similar edits.

File: portable/RVDS/ARM_CM4F/port.c [hunk 2]
```diff
@@ -468,36 +468,36 @@ __asm void xPortPendSVHandler( void )
     /* Is the task using the FPU context?  If so, push high vfp registers. */
     tst r14, #0x10
     it eq
-    vstmdbeq r0!, {s16-s31}
+    vstmdbeq r0 !, { s16 - s31 }
```
Comment (RichardBarry) [inline portable/RVDS/ARM_CM4F/port.c:471]: Likewise "r0!" was correct, it specifies writing back to the register.  Likewise for all similar edits.

**Review by RichardBarry — CHANGES_REQUESTED** [review]

**Review by aggarg — DISMISSED** [review]

**Comment by carlo-dev-git** [comment]
@aggarg Could you help me? Some checks were not successful:
```
Checking file: portable/RVDS/ARM_CM4F/portmacro.h...FAIL
* We do NOT require that all files contain the FreeRTOS File Header (copyright + license).      *
* Include the required updates to the '*_checker.py' script in your PR to make the check pass.  *
```

**Review by aggarg — APPROVED** [review]

**Review by RichardBarry — APPROVED** [review]

**Pull request merged**